manager-parent/manager-core/src/main/resources/db/migration/0.2/V0.2.1__create_v2_db.sql

```diff
@@ -1,66 +1,69 @@
-CREATE TABLE BRIDGE_V2 (
```

Comment:
revert the changes in this file before the merge otherwise you'll make flyway cry ;)